Sources/Controllers/ALKConversationViewController+TableView.swift

```diff
@@ -74,6 +74,16 @@ extension ALKConversationViewController: UITableViewDelegate, UITableViewDataSou
                 }
                 return cell
 
+            } else if let messageMetadata = message.metadata, let _ = messageMetadata["sourceURLs"], !message.isMyMessage {
```

Comment:
create a variable for this ` sourceURLs`

Reply:
added